mne/stats/parametric.py

```diff
@@ -7,22 +7,15 @@
 import numpy as np
 from ..fixes import matrix_rank
 from functools import reduce
-
-defaults_twoway_rm = {
-    'parse': {
-        'A': [0],
-        'B': [1],
-        'A+B': [0, 1],
-        'A:B': [2],
-        'A*B': [0, 1, 2]
-    },
-    'iter_contrasts': np.array([(1, 0, 1), (0, 1, 1), (1, 1, 1)])
-}
-
+from string import ascii_uppercase
```

Comment:
and actually this line and the `functools` line should go up above the `..fixes` line (general imports, then mne relative imports)